src/core/util.js

```diff
@@ -283,4 +285,221 @@ var parseTimeSeriesForC3 = function(graph_data) {
   };
 }
 
-module.exports = { parseDataForC3, parseTimeSeriesForC3, dataApiToC3 }
+/*
+    Takes a multistats object of the following form and 
+    1) flattens it, and
+    2) rounds numeric values 
+    for passing to the DataTable component for rendering:
+
+    {
+      "tasmin_Amon_CanESM2_historical_r1i1p1_19610101-19901231": 
+      {
+        "median": 278.34326171875,
+        "min": 225.05545043945312,
+        "units": "K",
+        "mean": 273.56732177734375,
+        "max": 303.601318359375,
+        "ncells": 8192,
+        "stdev": 22.509726901403784,
+        "run": "r1i1p1"
+      },
+      "tasmin_Amon_CanESM2_historical_r1i1p1_19710101-20001231": 
+      {
+        ...
+      }
+    };
+*/
+var parseBootstrapTableData = function(data) {
```

Comment:
You should be able to simplify this whole function by doing something like:
~~~js
return _.map(data, function(val, key) {
...
});